app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -7,20 +7,31 @@ package org.mozilla.focus.browser
 import android.content.Context
```

Comment:
Nice comment, and in the future, if you move a ton of code, that in and of itself should be a commit (even if it doesn't compile, but make a note in the commit message). That way, I don't need to re-review code that already is present and just being moved so I can hunt down the changes you made - I can just look at the subsequent commit.